doc/api/worker_threads.md

```diff
@@ -434,6 +434,16 @@ if (isMainThread) {
     process (such as `--title`) are not supported. If set, this will be provided
     as [`process.execArgv`][] inside the worker. By default, options will be
     inherited from the parent thread.
+  * `resourceLimits` {Object} An optional set of resource limits for the new
```

Comment:
Maybe we should name it `vmLimits`, or `vmSettings` (we could add more vm-related options there in the future)?

Or do we plan to add support of other types of resource limits here in the future?

Reply:
> Or do we plan to add support of other types of resource limits here in the future?

I wouldn’t be opposed to that, no.